lib/http/2/compressor.rb

```diff
@@ -359,15 +349,15 @@ def set_table_size(size)
       # @return [String] binary string
       def integer(i, n)
         limit = 2**n - 1
-        return [i].pack('C') if (i < limit)
+        return [i].pack('C') if i < limit
 
         bytes = []
-        bytes.push limit if !n.zero?
+        bytes.push limit unless n.zero?
```

Comment:
I prefer `unless` over `if negative-predicate` when it's postfix `if/unless`.
That style-guide looks good and well-accepted (though I disagree on some minor points).